pcsx2-qt/Settings/InterfaceSettingsWidget.cpp

```diff
@@ -34,6 +34,8 @@ static const char* THEME_NAMES[] = {
 	//: Ignore what Crowdin says in this string about "[Light]/[Dark]" being untouchable here, these are not variables in this case and must be translated.
 	QT_TRANSLATE_NOOP("InterfaceSettingsWidget", "Baby Pastel (Pink) [Light]"),
 	//: Ignore what Crowdin says in this string about "[Light]/[Dark]" being untouchable here, these are not variables in this case and must be translated.
+	QT_TRANSLATE_NOOP("InterfaceSettingsWidget", "Pizza Time! (Pizza) [Light]"),
```

Comment:
instead of pointlessly putting "(Pizza)" can you actually say what colours it roughly is :P Besides it looks more of a peach colour to me than pizza coloured.

Reply:
I actually see it as Brownish/Creamy White :P 
its kinda hard to nail down the exact color but i'll try to my best to describe it